api/services/S3Helper.js

```diff
@@ -10,10 +10,14 @@ const {
 
 const S3_DEFAULT_MAX_KEYS = 1000;
 
+const isLocal = process.env.FEATURE_LOCAL_S3_ENDPOINT;
+const endpoint = isLocal ? process.env.MINIO_ENDPOINT_URL : null;
+
 class S3Client {
   constructor(credentials) {
     this.bucket = credentials.bucket;
     this.client = new S3({
+      endpoint,
```

Comment:
What is provided here in deployed environments?

Reply:
Undefined. The endpoint url is only needed for S3 compliant services that aren't s3.